src/mgr/Mgr.cc

```diff
@@ -418,6 +421,15 @@ void Mgr::handle_osd_map()
   daemon_state.cull(CEPH_ENTITY_TYPE_OSD, names_exist);
 }
 
+void Mgr::handle_log(MLog *m)
+{
+  for (const auto &e : m->entries) {
+    std::ostringstream ss;
+    ss << e.stamp << " " << e.who.name << " " << e.prio << " " << e.msg;
```

Comment:
Not that you're asking for review, but, browsing this: ss is thrown away here, right?  Is this for gdb, or dead code, or?...